django-stubs/urls/resolvers.pyi

```diff
@@ -14,6 +14,7 @@ class ResolverMatch:
     namespace: str = ...
     view_name: str = ...
     route: str = ...
+    _func_path: str = ...
```

Comment:
We tend to use `_func_path: str` notation for things that don't have a class-level default value. Is this the case?

Reply:
Yes